executor/adapter.go

```diff
@@ -358,8 +358,16 @@ func (a *ExecStmt) buildExecutor(ctx context.Context) (Executor, error) {
 }
 
 func (a *ExecStmt) logSlowQuery(txnTS uint64, succ bool) {
+	level := log.GetLevel()
+	if level < log.WarnLevel {
```

Comment:
level < log.WarnLevel ? log.ErrorLevel is less than log.WarnLevel, so why return here?

Reply:
There is no need to print slow query log If log level is Error or Fatal.